omero/sysadmins/in-place-import.rst

```diff
@@ -4,7 +4,7 @@ In-place import
 ===============
 
 In-place import allows files which are already present on the server machine
```

Comment:
Rephrasing: In-place import allows image files that are accessible from the OMERO.server's filesystem...

(because they do not have to be stored on the server machine, but can be on any machine if mounted into the servers FS)